src/wobjectdefs.h

```diff
@@ -188,11 +188,11 @@ namespace binary {
 
 
 /** Compute the sum of many integers */
-template<typename... Args> constexpr void noOp(Args...) {}
+template<typename T> constexpr void ordered(std::initializer_list<T>) {}
 template<typename... Args>
 constexpr int sums(Args... args) {
     auto i = int{};
-    noOp((i += args, 0)...);
+    ordered<int>({(i += args, 0)...});
```

Comment:
Could this be done with  simply  `std::initializer_list<int>{ (i += args, 0)...  }` maybe, so avoiding the helper.

Reply:
Yes, but you would also have to cast it to void or the compiler might complain that the result is not used.

But we would hide intend of this hack. Therefore I like the method named with the intention of ordering the expansion.